defaults/main.yml

```diff
@@ -81,6 +81,11 @@ devture_woodpecker_ci_server_config_grpc_addr_port: 9000
 # See: https://woodpecker-ci.org/docs/administration/server-config#woodpecker_grpc_addr
 devture_woodpecker_ci_server_config_grpc_addr: ":{{ devture_woodpecker_ci_server_config_grpc_addr_port }}"
 
+# devture_woodpecker_ci_server_config_grpc_expose_externally controls whether
+# to expose the gRPC port externally.  Set this to true if you plan to run Woodpecker agents
+# on separate machines/networks.
+devture_woodpecker_ci_server_config_grpc_expose_externally: false
```

Comment:
Why not call this `devture_woodpecker_ci_server_container_labels_traefik_grpc_enabled` and move it down next to the other label variables?

Reply:
I think it's clearer to name the variable in a way that indicates that it will be exposing the service to the internet when set to `true`, but I'm open to rename it as suggested if you think it's better.

Reply:
All Traefik labels are for exposing services to the internet (unless the labels are for some internal entrypoint, but we have no such entrypoints in this playbook and its roles).

I think at least a few other roles have the ability to enable/disable certain labels with an `_enabled` variable, so this is quite consistent among our roles. Here's an example from the Keycloak role: https://github.com/mother-of-all-self-hosting/ansible-role-keycloak/blob/3b59862aaa2ffbeb06f395574e334f1f64ee37c3/templates/labels.j2#L75-L113

Reply:
ACK, thanks.  Fixed in the last force-push.